Add option to or not to end url with slash

imjeffparedes opened the pull request:
Add option to end url with slash or not

Commit:
Update lib/index.js

Add option to end url with slash or not

imjeffparedes committed:
Merge branch 'master' into add-url-slash-option

imjeffparedes commented:
Fix for endpoint not working due to trailing slash problem.

Originally caused by:

[#22](https://github.com/IonicaBizau/coins-ph/issues/22) sellorder wans't working
GET https://coins.ph/api/v2/sellorder/4dbf5b20/?sellorder_id=4dbf5b20

[#23](https://github.com/IonicaBizau/coins-ph/pull/23) was merge to fix it but broke other endpoints:
GET https://coins.ph/d/api/payin-outlets?region=PH
GET https://coins.ph/api/v3/crypto-accounts?currency=PBTC

@iceniveth

iceniveth commented on lib/index.js:
```diff
@@ -459,7 +460,8 @@ module.exports = class Coins {
           , version = options.version || "d/api"
           , qs = querystring.stringify(params)
-          , url = this.host + version + "/" + _url + "/" + (qs ? "?" + qs : "")
+          , endWithSlash = options.endWithSlash || true
```
I was wrong with this condition because this would always be true. 😅

It should be: `, endWithSlash = options.endWithSlash == null ? true : options.endWithSlash`

imjeffparedes replied:
Oh right. Did not noticed that. I'll try another better approach.

imjeffparedes replied:
@iceniveth I indicate to remove trailing slash option instead. By default, every url should have a trailing slash at the end of the url.

imjeffparedes committed:
Indicate to remove trailing slash option instead

endWithSlash option will always be true. Indicate to remove trailing slash option instead.
